pro/src/core/OfferEducational/adapters/deleteActiveBookingsAdapter.ts

```diff
@@ -0,0 +1,35 @@
+import * as pcapi from 'repository/pcapi/pcapi'
+
+type IPayloadSuccess = null
+type IPayloadFailure = null
+type DeleteActiveBookingsAdapter = Adapter<
+  { offerId: string },
+  IPayloadSuccess,
+  IPayloadFailure
+>
+
+export const deleteActiveBookingsAdapter: DeleteActiveBookingsAdapter = async ({
+  offerId,
+}) => {
+  try {
+    await pcapi.cancelEducationalBooking(offerId)
+
+    /* @debt bugRisk "Gaël: we can't be sure this way that the stock is really booked, it can also be USED or CANCELED"*/
+    return {
+      isOk: true,
+      message:
+        'Les réservations et pré-reservations sur cette offre ont été annulés avec succès, votre offre sera à nouveau visible par tous sur adage',
```

Comment:
(typo)

```suggestion
        'Les réservations et pré-reservations sur cette offre ont été annulées avec succès, votre offre sera à nouveau visible par tous sur Adage',
```

... voire même "ADAGE" ? Je ne sais pas du tout ! :)  Il n'y a qu'une seule occurrence dans le portail pro pour l'instant et elle est en majuscule.